staging/src/k8s.io/apiserver/pkg/util/flowcontrol/fairqueuing/queueset/queueset.go

```diff
@@ -186,14 +209,26 @@ const (
 // irrelevant.
 func (qs *queueSet) Wait(ctx context.Context, hashValue uint64, descr1, descr2 interface{}) (tryAnother, execute bool, afterExecution func()) {
 	var req *request
-	decision := func() string {
+	decision := func() decision {
```

Comment:
nit: this variable name makes the switch statements below confusing, can we change it to "finalDecision" or something like that?

Reply:
OK, I will give the type a more descriptive name.